libstuff/libstuff.h

```diff
@@ -272,31 +266,20 @@ void SLogStackTrace();
 // --------------------------------------------------------------------------
 // Thread stuff
 // --------------------------------------------------------------------------
-// Light wrapper around thread functions
-void* SThreadOpen(void (*proc)(void* procData), void* procData, const string& threadName = "", size_t stackSize = 0);
-void SThreadClose(void* thread);
-void SThreadSleep(uint64_t delay);
 
-// Thread local storage
-struct SThreadLocalStorage {
-    // Attributes
-    void (*proc)(void* procData);
-    void* procData;
-    string name;
-    string logPrefix;
-    SData data;
-};
-
-SThreadLocalStorage* SThreadGetLocalStorage();
+// Each thread gets its own thread-local log prefix.
+extern thread_local string SThreadLogPrefix;
+extern thread_local string SThreadLogName;
```

Comment:
The convention we've used for globals is to prefix with `g_`, so this would be `g_SThreadLogPrefix` -- can you rename?